src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -499,7 +493,7 @@ const Eligibility: React.FC<EligibilityProps> = ({
           return item
         })}
       </MobileEligibility>
-      <DesktopEligibility sx={{ display: ['none', 'block'] }} hover={hover}>
+      <DesktopEligibility className="desktop-eligibility">
```

Comment:
how does this limit by device?, shouldnt be on mobile

Reply:
it's above in the wrapper